StripeCore/StripeCore/Source/DownloadManager/DownloadManager.swift

```diff
@@ -93,15 +93,16 @@ extension DownloadManager {
         let blockingDownloadSemaphore = DispatchSemaphore(value: 0)
 
         let urlRequest = URLRequest(url: url, cachePolicy: .useProtocolCachePolicy)
-        let task = self.session.downloadTask(with: url) { tempURL, response, _ in
+        let task = self.session.downloadTask(with: url) { tempURL, response, error in
             guard let tempURL = tempURL,
                 let response = response,
                 let data = self.getDataFromURL(tempURL),
                 let image = self.persistToMemory(data, forImageName: imageName)
             else {
-                let errorAnalytic = ErrorAnalytic(event: .unexpectedStripeCoreDownloadManagerError,
-                                                  error: Error.downloadSyncFailure,
-                                                  additionalNonPIIParams: ["url": url.absoluteString])
+                let errorAnalytic = ErrorAnalytic(event: .stripeCoreDownloadManagerError,
+                                                  error: Error.downloadAsyncFailure,
+                                                  additionalNonPIIParams: ["url": url.absoluteString,
+                                                                           "url_error_code": (error as? NSError)?.code ?? "none",])
```

Comment:
We will create an alert which always alerts on a "none" "url_error_code", as that means we got a valid response but failed to create an image from it.